packages/main/src/TreeListItem.js

```diff
@@ -170,6 +170,18 @@ const metadata = {
 			noAttribute: true,
 		},
 
+		/**
+		 * Defines if the item should be collapsible or not.
```

Comment:
As I see this property is not used when creating the `ui5-li-tree` instances in the `Tree.hbs`, we can remove the definition here. Using it in the `get requiresToggleButton()` is enough to hide the toggle button.